Applications/Utils/Tests.cmake

```diff
@@ -331,3 +331,16 @@ if(OGS_USE_NETCDF)
         slim_100897_198.vtu slim_100897_198.vtu NO NO 1e-16 0
     )
 endif()
+
+if(OGS_BUILD_GUI)
```

Comment:
why gui? It's a command line tool...

Reply:
Lars added the ```OGS_USE_GUI```-tag in the cmake-file so that the ghost data filter would be linked. Since it's added there, I figured it should be added here, too?